asmcli/asmcli.sh

```diff
@@ -1973,12 +1973,17 @@ is_membership_crd_installed() {
 populate_environ_info() {
   local ENVIRON_PROJECT_ID; ENVIRON_PROJECT_ID="$(context_get-option "ENVIRON_PROJECT_ID")"
   local HUB_MEMBERSHIP_ID; HUB_MEMBERSHIP_ID="$(context_get-option "HUB_MEMBERSHIP_ID")"
+  local HUB_IDP_URL; HUB_IDP_URL="$(context_get-option "HUB_IDP_URL")"
```

Comment:
this will work since we check `-n` but do we want to eventually add this to the context file? Since we will have a json file describing the schema of the context file in the end we could add this to the context file just in case we forgot?